vpn/src/main/java/com/duckduckgo/mobile/android/vpn/bugreport/AppHealthStateCollector.kt

```diff
@@ -32,7 +32,7 @@ class AppHealthStateCollector @Inject constructor(
     override val collectorName: String = "latestAppBadHealth"
 
     override suspend fun collectVpnRelatedState(appPackageId: String?): JSONObject {
-        val healthState = appHealthDatabase.appHealthDao().latestHealthState()
+        val healthState = appHealthDatabase.appHealthDao().remove()
```

Comment:
maybe I miss the point, but if you remove something is it meant to return a value?

Reply:
I took the same approach as what eg. `LRUCache` API. Where remove removes from the cache and gives you the removed item 🤷